packages/vite/src/node/plugins/importMetaGlob.ts

```diff
@@ -14,6 +14,7 @@ import MagicString from 'magic-string'
 import fg from 'fast-glob'
 import { stringifyQuery } from 'ufo'
 import type { GeneralImportGlobOptions } from 'types/importGlob'
+import type { TemplateLiteral } from '@babel/types'
```

Comment:
Should this be part of the `estree` import instead of `@babel/types`?

Reply:
Yes, thank you for reminding me.